MoreLinq.Test/HasDuplicatesTest.cs

```diff
@@ -0,0 +1,84 @@
+namespace MoreLinq.Test
+{
+    using System.Collections.Generic;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class HasDuplicatesTest
+    {
+        [Test]
+        public void When_Asking_For_Duplicates_On_IEnumerable_Without_Duplicates_Then_False_Is_Returned()
+        {
+            var stringArray = new[]
+            {
+            "FirstElement",
```

Comment:
nit: incorrect indentation - array initialization elements should be one-tab further than the braces.